src/components/ForecastScreen.js

```diff
@@ -7,46 +6,56 @@ import {
   InteractionManager,
   ActivityIndicator,
 } from 'react-native';
-import PropTypes from 'prop-types';
 import { autobind } from 'core-decorators';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import ForecastListItem from './ForecastListItem';
-import * as weatherService from '../services/weather';
-import * as navigationService from '../services/navigation';
+import { resetCurrentCity, fetchForcastForCity, resetForcast } from '../redux/actions';
+import { getCurrentCity, getForcast } from '../redux/selectors';
 
 class ForecastScreen extends Component {
+  static navigationOptions = {
+    title: 'Grad',
```

Comment:
how about reading city title from navigation props the way we talked about? :)
We can discuss it again if you want.